packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -45,39 +65,46 @@
   }
 
   .heading {
-    @apply pointer-events-none
-      break-words;
-    color: var(--calcite-color-text-2);
+    color: var(--calcite-tile-heading-text-color);
     font-size: var(--calcite-font-size--1);
     font-weight: 500;
     line-height: 1.20313rem;
```

Comment:
where is this line height coming from?

Reply:
Also from the Figma file.  These are copied verbatim from the styles panel of the corresponding text elements:

![image](https://github.com/Esri/calcite-design-system/assets/821864/82c0e8a0-0cb5-48d4-aab0-dfc1de964fe4)

Reply:
@SkyeSeitz we haven't set Semantic Tokens in Tile Component yet right? I think this needs to be replaced with a semantic line-height token. Pick the one closest to this value.